src/wp-admin/includes/class-bulk-plugin-upgrader-skin.php

```diff
@@ -16,7 +16,16 @@
  * @see Bulk_Upgrader_Skin
  */
 class Bulk_Plugin_Upgrader_Skin extends Bulk_Upgrader_Skin {
-	public $plugin_info = array(); // Plugin_Upgrader::bulk_upgrade() will fill this in.
+
+	/**
+	 * Plugin info.
+	 *
+	 * The `Plugin_Upgrader::bulk_upgrade()` will fill this in with info
+	 * retrieved from the `Plugin_Upgrader::plugin_info()` method.
+	 *
+	 * @var string|false The full path to the main plugin file, or false.
+	 */
+	public $plugin_info = array();
```

Comment:
This is initialized as `array` but documented as `string|false`. Should this be `string[]|false`? Same question for the others in the PR.

Reply:
Argh.... You're right, in part the default values are wrong, in part the docblocks are wrong.

I took a shortcut here and basically just copied the information which was already in place in the two classes which _did_ have the properties declared.

I have now verified it again and for the `$plugin_info` properties, the docblock info was wrong, while for the `$theme_info` properties, the default value was wrong.

I have fixed this now, both for those classes which already contained the properties, as well as in the new property declarations being added now.